bigquery/README.md

```diff
@@ -20,8 +20,21 @@ if you haven't done so already.
   [Overview](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview)
```

Comment:
Since docker compose is for running multiple containers, I still think it makes sense to have a docker-compose for elasticsearch + indexer. Running the indexer by itself can be a normal Dockerfile.

Volumes should be supported in normal docker: https://docs.docker.com/storage/volumes/

Reply:
As discussed, put elasticsearch back into docker-compose.yml. Tested:
- Elasticsearch from other repo, default dataset
- Elasticsearch from other repo, not default dataset
- Elasticsearch from this repo, not default dataset